### PR TITLE
Update service discovery references

### PR DESCRIPTION
**Pull request by ersonp**
Did you run `make format && make check`?

Fixes #	

 Changes:	
- Change `proxyDiscovery` to `serviceDiscovery`

How to test this PR:

### CODE REVIEW

File: pkg/visor/visorconfig/v1.go [hunk 1]
```diff
@@ -100,15 +100,15 @@ type V1UptimeTracker struct {
 	Addr string `json:"addr"`
 }
 
-// V1AppDisc configures Skywire App Discovery Clients.
-type V1AppDisc struct {
+// V1Disc configures Skywire App Discovery Clients.
+type V1Disc struct {
```
Comment (jdknives) [inline pkg/visor/visorconfig/v1.go]: `ServiceDisc` maybe better? We have three discoveries at this point overall.

**Review by jdknives — COMMENTED** [review]
Should we rename the package `appdisc` as well? Or is just responsible for updating app statuses in service discovery? Also do we need to define a config migration, right ? Also, please add changes to changelog. Will need to start doing that more diligently.

File: pkg/visor/visorconfig/v1.go (hunk 1 above)
Comment (ersonp) [reply pkg/visor/visorconfig/v1.go]: Okay

**Comment by ersonp** [comment]
- We can name `appdisc` to `updatedisc` instead since that is what it's doing since `appdisc` can be confusing. 
- The `updateUrls` in `parse` takes care of the migration. Only thing is that the URL is changed to `http://sd.skycoin.com` with it.
- Will add all the changes of config `v1.1.0` to Changelog. Also, I will need some inputs for all the changes introduced in the current version of `skywire@develop`.

**Review by jdknives — APPROVED** [review]

**Pull request merged**